cli-plugins/insight/triaging-vulnerabilities.hbs.md

```diff
@@ -0,0 +1,111 @@
+# Triaging vulnerabilities
+
+This topic describes how to add analyses to any vulnerability associated to a workload
+in the Supply Chain Security Tools - Store. Keep in mind this is an experimental feature, and the
+API is prone to changes in subsequent releases.
+
+## <a id='triage-description'></a>What is triaging?
+
+Vulnerability analysis, triaging or triage is the process of evaluating a reported vulnerability in
+order to determine an effective remediation plan. It is through triage that app teams are able to
+generate useful insights about the vulnerabilities in their software so that they can make the right
+decisions about when and how to mitigate them. The current implementation of triage follows
+[CycloneDX's Vulnerability Exploitability eXchange (VEX)](https://cyclonedx.org/capabilities/vex/)
+specification, and is designed specifically to work with Tanzu workloads.
+
```

Comment:
This could be a good place to add a link to the data model doc you wrote

Reply:
Added